### DISCUSSION TITLE
Adding additional Node to an existing 1-Node Cluster

### DISCUSSION THREAD

**Discussion by oiBio** [ravendb/ravendb#13404] [Q&A] [answered]
i'am struggeling to setup a cluster from my existing 1-Node-Cluster. We do have a running cluster with serveral databases and now need to add nodes to create a "real" cluster.

in the initial setup we used the lets encrypt setup and only added one node. 
how can i add new nodes to this cluster? When i use the install wizard on the new node and choose continue cluster setup i can only choos the existing node A, but cant add new ones.

i tried many things but it always failed. what do to in this case? is it possible to add another node with lets encrypt?

**Comment by ayende** [comment, marked as answer]
You need to start by setting up a new node, for that, you'll need to do the following.

* Copy the `*.pfx` and `settings.json` files from the current node to the new node.
* Update the `ServerUrl` / `PublicServerUrl` to point to the new node
* Update DNS, etc if needed

Start the RavenDB service with on the new node. At this point, it is running in passive mode, it isn't joined to the cluster.

On the existing node, go the Manage > Cluster and add the node.

That should be it.

**Reply by oiBio** [reply]
Hi Oren, thank you for your reply. 
How do i update the DNS for Lets Encrypt for the new node?

https://a.FOO.development.run/          <- was registred before
https://b.FOO.development.run/          <- new Node on new Server with another IP

**Reply by ayende** [reply]
You can go here, put the license key for `FOO` domain and then you can edit the DNS.
Note that there is 30 minutes expiration on the DNS entries.

**Reply by DejanMilicic** [reply]
https://customers.ravendb.net/

**Reply by oiBio** [reply]
Perfect thank you!
Our of curiosity, is this documenten in the docs or am i just blind?

**Reply by ayende** [reply]
You aren't blind, I talked about this here: https://ayende.com/blog/186309-C/ravendb-customers-portal

But it isn't in the official docs, for some reason. https://issues.hibernatingrhinos.com/issue/RDoc-2048
